index.html

```diff
@@ -17,12 +17,21 @@
 			</div>
 		</div>
 		<div class="Grid-logoWrapper"><img src="./assets/logo.png" alt="NK Flohmarkt am Kranoldplatz"></div>
-		<div class="Grid-description">Neukölln's neueur Sonntags Flohmarkt<br>beggint 9. Juni & jeden zweiten Sonntag</div>
-		<div class="Grid-booking">Stand buchen</div>
-		<div class="Grid-calendar">nächster Termin:<br>9.<br>Juni</div>
-		<div class="Grid-map">map</div>
-		<a class="Grid-infoLink" href="/info">Info</a>
-		<div class="Grid-video">lorem ipsum</div>
+		<div class="Grid-description">
+			<p class="Grid-description-text">Neukölln's neueur Sonntags Flohmarkt begint 9. Juni & jeden zweiten Sonntag</p>
+		</div>
+		<div class="Grid-booking">
+			<p class="Grid-booking-text">Stand buchen </p>
+		</div>
+		<div class="Grid-calendar">
+			<p class="Grid-calendar-text"> Nächster Termin: 9. Juni</p>
+		</div>
+		<div class="Grid-map"></div>
+		<a class="Grid-infoLink" href="/info">
+			<p class="Grid-infoLink-text">Info</p>
+		</a>
+		<div class="Grid-video">
+			<p class="Grid-video-text">Performace at 16PM</p>
```

Comment:
a thought here;

when you do `.Grid-booking-text` etc., it is very explicit. ONLY this booking text should look like that. If that's the case, fine - but consider `.BigText` and `.SmallText` or `.Subtitle` etc. That way they are not conceptually tied with the `.Grid`.

It's good to be able to move things around. If the grid and what's inside the grid is separated, it might make it easier down the road.